docs/GL5/SRI/Indexation/cheat-sheet.md

```diff
@@ -8,15 +8,15 @@ authors:
 
 Author [@medbensalah](https://github.com/medbensalah)
 
-## Indexation
+# Indexation
```

Comment:
Only one H1 is allowed.
H1 must be the name of the document

Reply:
LGTM 👍